lib/lotus/validations/attribute.rb

```diff
@@ -200,8 +200,9 @@ def size
       def nested
         _validate(__method__) do |validator|
           errors = value.validate
-          unless errors.empty?
-            @errors.set @name, errors
+          errors.each do |error|
+            new_error = Error.new(error.attribute, error.validation, error.expected, error.actual, @name)
```

Comment:
4 params is abit hard to follow, keyed params would be great for this constructor.